diff -cr linux-4.13.9/drivers/hid/usbhid/hid-core.c linux-4.13.9-patched/drivers/hid/usbhid/hid-core.c
*** linux-4.13.9/drivers/hid/usbhid/hid-core.c	2017-10-21 16:55:07.000000000 +0100
--- linux-4.13.9-patched/drivers/hid/usbhid/hid-core.c	2017-10-26 22:15:57.063151305 +0100
***************
*** 56,61 ****
--- 56,65 ----
  module_param_named(jspoll, hid_jspoll_interval, uint, 0644);
  MODULE_PARM_DESC(jspoll, "Polling interval of joysticks");
  
+ static unsigned int hid_otherhid_interval;
+ module_param_named(otherhidpoll, hid_otherhid_interval, uint, 0644);
+ MODULE_PARM_DESC(otherhidpoll, "Polling interval of all non-mice/joystick hid devices");
+ 
  static unsigned int ignoreled;
  module_param_named(ignoreled, ignoreled, uint, 0644);
  MODULE_PARM_DESC(ignoreled, "Autosuspend with active leds");
***************
*** 1105,1110 ****
--- 1109,1118 ----
  			if (hid_jspoll_interval > 0)
  				interval = hid_jspoll_interval;
  			break;
+ 		default:
+ 			if (hid_otherhid_interval > 0)
+ 				interval = hid_otherhid_interval;
+ 			break;
  		}
  
  		ret = -ENOMEM;
